docs/preview/features/logging.md

```diff
@@ -255,6 +257,75 @@ So the resulting log message becomes:
 
 `HTTP Request POST http://localhost:5000/weatherforecast completed with 200 in 00:00:00.0191554 at 03/23/2020 10:12:55 +00:00 - (Context: [X-Api-Key,])`
 
+#### Tracked HTTP status codes
+
+By default, all the responded HTTP status codes will be tracked but this can be altered according to your chosing.
+
+Consider the following API controller. When we configure request tracking, both the `400 BadRequest` response as the `201 Created` response will be tracked.
+
+```csharp
+[ApiController]
+public class OrderController : ControllerBase
+{
+    [HttpPost]
+    public IActionResult Create([FromBody] Order order)
+    {
+        if (order.Id is null)
+        {
+            // Request tracking will happen on this response.
+            return BadRequest("Order ID should be filled out");
+        }
+
+        // Request tracking will happend on this response.
+        return Created($"/orders/{order.Id}", order);
+    }
+}
+```
+
+Let's change the request tracking to only track successful '201 Created' responses.
+This can be changed via the options:
+
+```csharp
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+
+public class Startup
+{
+    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    {
+        app.UseRequestTracking(options => options.TrackedStatusCodes.Add(HttpStatusCode.Created));
+    }
+}
+```
+
+This means that every endpoint will only track `201 Created` responses. Changing this in the options usually for when you want to straightline your entire application and only allow a certain set of status codes to be tracked.
+More grainer control can be achieved via placing an attribute on either the controller's class definition or the endpoint method:
+
+```csharp
+using Arcus.WebApi.Logging;
+
+[ApiController]
+public class OrderController : ControllerBase
+{
+    [HttpPost]
+    [RequestTracking(HttpStatusCode.Created)]
```

Comment:
Would love to support this:
```suggestion
    [RequestTracking(HttpStatusCode.Created, HttpStatusCode.OK)]
```

Or define a range from 200 -> 299.

Reply:
Ah, that would be better. Now it's supported for multiple ones that you can inside one attribute but we could use it indeed for ranges. That would be better maybe.

Reply:
But let's do that in another issue too. Now it already supports this syntax but it takes the two status codes separatly in. Maybe for ranges, we should come up with our own enumeration or let it specify as integers.

Reply:
I'll remove the `params` definition for now so we can add these ranges later.

Reply:
Ok, removed these `params`. I'll create an issue for the ranges.

Reply:
Done https://github.com/arcus-azure/arcus.webapi/issues/216, resolving.